control-plane/pkg/reconciler/channel/v2/controllerv2.go

```diff
@@ -138,6 +155,14 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *conf
 			corev1.SchemeGroupVersion.WithKind("ConfigMap"),
 		),
 	))
+
+	parts := strings.Split(eventingtls.TrustBundleLabelSelector, "=")
+
+	configmapInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: kafka.FilterWithLabel(parts[0], parts[1]),
+		Handler:    controller.HandleAll(globalResync),
+	})
```

Comment:
There are key and value as constant so we don't need to split

https://github.com/knative/eventing/blob/4ba426cd4305e014421402a2046f34b7bb9bf5e9/pkg/eventingtls/trust_bundle.go#L38-L40